app/controllers/data_services_controller.rb

```diff
@@ -16,7 +16,7 @@ def data_services
     @data_services = if params[:query].blank? && params[:filters].blank?
                        DataService.includes(:organisation).all.order('name ASC')
                      else
-                       SearchService.call(query: params[:query], filters: params[:filters])
+                       FilterService.call(query: params[:query], filters: params[:filters])
```

Comment:
I think we can remove this whole `if ... else` from the controller and just call the Filter/Search service (however we prefer to call it) here, so the controller doesn't contain any of this logic and instead it's all happening in the service. Just need to make sure we carry over anything that's in here into the service if it's not already there (although I think we're already checking the presence of the params in the service).

Reply:
Logic has been moved to the service :)